index.js

```diff
@@ -130,9 +156,17 @@ module.exports = stream =>
         .set(data.rollingCountThreadPoolRejected);
       rollingCountTimeout.labels(name).set(data.rollingCountTimeout);
 
+      // Requires brakes version 2.7.0 +
       currentConcurrentExecutionCount
         .labels(name)
-        .set(data.currentConcurrentExecutionCount);
+        .set(data.currentConcurrentExecutionCount || 0);
+      cummulativeCountTimeout.labels(name).inc(data.countTimeoutDeriv || 0);
```

Comment:
is this correct?

if the counter is `4`, then `inc(5)` it'll be set to `9`, no?

Reply:
That was a mistake, fixed it with the commit #bb48710

Reply:
I don't see how that changed?

Reply:
currentConcurrentExecutionCount is a Gauge and we only receive the current state of the buckets from brakes here. This metric did not change in my PR (added || 0 by mistake).

For the other counters: brakes exposes the total counter and the change since the last measurement. We can therefore use countTimeout (total counter of timeouts) or countTimeoutDeriv. The latter shows the change since the last measurement. When the counter was 4 and changed to 5 the deriv counter would for the same two measurements 4 and 1 (as it just increased by one). We then use the inc operation to incrase the timeout counter. 

I would have liked to use the set operation, but the Counter implement of prom-client does not expose such an operation for Counters (only for Gauges).

Reply:
right, perfect! thanks for explaining